### PR TITLE
V3: Fix several case table bugs

### PR DESCRIPTION
**Pull request by kswenson**
- horizontal scroll bar not visible
- column headers no longer bold
- no row selection highlighting on odd rows
- clicking on row sometimes scrolls unnecessarily
- selecting case in graph doesn't always scroll corresponding row into view

### CODE REVIEW

**Review by bfinzer — APPROVED** [review]
👍🏻 LGTM

File: v3/src/components/case-table/case-table.scss [hunk 1]
```diff
@@ -14,6 +16,7 @@
         .codap-column-header-content {
           width: 100%;
           height: 100%;
+          font-weight: bold !important;
```
Comment (eireland) [inline v3/src/components/case-table/case-table.scss:19]: Is this for the drag overlay? We are already specifying `fontWeight="bold"` in inline styling.

**Review by eireland — APPROVED** [review]
looks good except for maybe redundant styling?

File: v3/src/components/case-table/case-table.scss (hunk 1 above)
Comment (kswenson) [reply v3/src/components/case-table/case-table.scss:19]: Good catch -- probably redundant. I'll check and remove it if so.

**Pull request merged**